autoload/vaffle/buffer.vim

```diff
@@ -30,6 +30,7 @@ function! s:set_up_default_mappings() abort
   call s:map_default('n', 'l',       'open-current',     '<buffer> <silent>')
   call s:map_default('n', 't',       'open-current-tab', '<buffer> <nowait> <silent>')
   " Misc
+  call s:map_default('n', 'c',       'chdir-to-here',    '<buffer> <silent>')
```

Comment:
I cannot judge `'c'` is suitable for the feature at this time. Would you remove it from the default mappings?

Reply:
OK, I removed it.